### PR TITLE
Add init support of orgmode document type on file view and readme

### PR DESCRIPTION
**Pull request by lunny**
will be part of #374 and replace #1527

### CODE REVIEW

**Comment by lunny** [comment]
CI failed seems unrelated.

File: modules/markup/orgmode/orgmode_test.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	"testing"
+
+	"code.gitea.io/gitea/modules/markup"
+	"code.gitea.io/gitea/modules/setting"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const AppURL = "http://localhost:3000/"
+const Repo = "gogits/gogs"
```
Comment (jonasfranz) [inline modules/markup/orgmode/orgmode_test.go:18]: Why `gogits/gogs`?

Comment (lunny) [reply modules/markup/orgmode/orgmode_test.go:18]: No reason. Just copied from markdown test.

File: main.go [hunk 2]
```diff
@@ -14,6 +14,10 @@ import (
 	"code.gitea.io/gitea/modules/log"
 	"code.gitea.io/gitea/modules/setting"
 	"github.com/urfave/cli"
+
+	// register supported doc types
+	_ "code.gitea.io/gitea/modules/markup/markdown"
```
Comment (ethantkoenig) [inline main.go]: Import order

**Review by ethantkoenig — CHANGES_REQUESTED** [review]

File: main.go (hunk 2 above)
Comment (lunny) [reply main.go]: done.

**Review by ethantkoenig — APPROVED** [review]

**Review by sapk — APPROVED** [review]

**Comment by sapk** [comment]
LGTM

**Comment by lafriks** [comment]
LGTM

File: modules/markup/orgmode/orgmode_test.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+const AppSubURL = AppURL + Repo + "/"
+
+func TestRender_StandardLinks(t *testing.T) {
+	setting.AppURL = AppURL
+	setting.AppSubURL = AppSubURL
+
+	test := func(input, expected string) {
+		buffer := RenderString(input, setting.AppSubURL, nil, false)
+		assert.Equal(t, strings.TrimSpace(expected), strings.TrimSpace(string(buffer)))
```
Comment (ethantkoenig) [inline modules/markup/orgmode/orgmode_test.go]: nit: unnecessary cast of `buffer`

File: modules/markup/orgmode/orgmode_test.go [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+		`<p><a href="`+lnk+`" title="WikiPage">WikiPage</a></p>`)
+}
+
+func TestRender_Images(t *testing.T) {
+	setting.AppURL = AppURL
+	setting.AppSubURL = AppSubURL
+
+	test := func(input, expected string) {
+		buffer := RenderString(input, setting.AppSubURL, nil, false)
+		assert.Equal(t, strings.TrimSpace(expected), strings.TrimSpace(string(buffer)))
```
Comment (ethantkoenig) [inline modules/markup/orgmode/orgmode_test.go]: nit: unnecessary cast of buffer

**Review by ethantkoenig — APPROVED** [review]
LGTM, just two nits

**Comment by lunny** [comment]
@ethantkoenig done.

**Pull request merged**